examples/fodo_rf/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _examples-fodo:
```

Comment:
Please add this new file in `docs/source/usage/examples.rst`, so that the table-of-contents picks it up when it builds the HTML/LaTeX page :)

Reply:
part for #114 :heavy_check_mark: